src/common/utils/model.ts

```diff
@@ -98,49 +98,24 @@ export function fetchAllStateProjections(snapshotUrl: string | null = null) {
   return cachedStatesProjections[key];
 }
 
-/** Returns an array of `Projections` instances for all counties. */
 const cachedCountiesProjections: { [key: string]: Promise<Projections[]> } = {};
-export function fetchAllCountyProjections(
-  snapshotUrl: string | null = null,
-  queryByState: boolean = true,
-) {
-  if (queryByState) {
-    return fetchAllCountyProjectionsByState();
-  }
-
-  snapshotUrl = snapshotUrl || getSnapshotUrlOverride();
+export function fetchAllCountyProjections(snapshotUrl: string | null = null) {
   async function fetch() {
-    const all = await new Api(snapshotUrl).fetchAggregatedSummaryWithTimeseries(
-      APIRegionSubPath.COUNTIES,
+    // Query counties for states individually as the entire counties.timeseries.json is too large
+    // to be parsed by node.
+    const allProjections = await Promise.all(
+      regions.states.map(
+        async (state: State) => await fetchCountyProjectionsForState(state),
+      ),
     );
-    return all
-      .filter(summaryWithTimeseries => {
-        // We don't want to return county projections for counties that we
-        // don't have in the regions instance
-        const region = regions.findByFipsCode(summaryWithTimeseries.fips);
-        return region instanceof County && Boolean(region.state);
-      })
-      .map(summaryWithTimeseries => {
-        const region = regions.findByFipsCodeStrict(summaryWithTimeseries.fips);
-        return new Projections(summaryWithTimeseries, region);
-      });
+    return allProjections.flatMap(obj => obj);
```

Comment:
Looks like you can just do `.flat()` ?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat